gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -153,6 +151,26 @@ def _bigquery_to_vcf_shards(
          | vcfio.WriteVcfDataLines())
 
 
+def _form_customized_query(known_args):
+  # type: (argparse.Namespace) -> str
+  """Returns a customized query for the interested regions."""
+  base_query = _BASE_QUERY_TEMPLATE.format(INPUT_TABLE='.'.join(
+      bigquery_util.parse_table_reference(known_args.input_table)))
+  if not known_args.genomic_regions:
+    return base_query
+
+  conditions = []
+  for region in known_args.genomic_regions:
+    ref, start, end = genomic_region_parser.parse_genomic_region(region)
+
+    conditions.append('{}=\'{}\' AND {}>={} AND {}<={}'.format(
```

Comment:
nit: could you please surround this query with brackets (i realize that ANDs are evaluated before ORs, but it makes it cleaner to read/debug later).